sdcm/utils/docker_utils.py

```diff
@@ -399,18 +401,29 @@ def build_container_image(cls, instance: INodeWithContainerManager, name: str, *
             getattr(instance, f"{name.family}_container_image_build_args", cls._build_args)(**extra_build_args))
 
         LOGGER.debug("Build arguments for Docker image `%s':\n%s,", image_tag, pformat(build_args, indent=8))
-        try:
-            image, logs = docker_client.images.build(tag=image_tag, **dockerfile_args, **build_args)
-        except BuildError as docker_build_error:
-            LOGGER.critical("Build log:\n%s",
-                            "\n".join(item['stream'] for item in docker_build_error.build_log if "stream" in item))
-            raise
-
-        LOGGER.debug(">>> Build log for Docker image `%s': >>>", image_tag)
-        for entry in logs:
-            if "stream" in entry:
-                LOGGER.debug(entry["stream"].rstrip())
-        LOGGER.debug("<<<")
+
+        resp = docker_client.api.build(tag=image_tag, **dockerfile_args, **build_args)
```

Comment:
please add a note that this code was taken from docker module, so in case it fails (e.g. regexp/api changes) we know where to copy&adapt from again.

Reply:
@fruch please, move the ported code to a separate function and call it here for ease of future updates.